src/text/line.rs

```diff
@@ -218,6 +218,57 @@ impl<'a> Line<'a> {
         }
     }
 
+    /// Left-aligns this line of text.
+    ///
+    /// Convenience shortcut for `Line::alignment(Alignment::Left)`.
```

Comment:
Can you please add something like the following: "Setting the alignment of a Line generally overrides any alignment set on the Text or Widget that contains the Line." and also to the alignment setter. And perhaps something similar to noting the default? There's probably a way to communicate this more concisely in a single sentence too.

I say generally, because third party custom widgets can render however they want, but our widgets will try to adhere to this when possible.

Reply:
How about this?

> /// Setting the alignment of a Line generally overrides the alignment of its 
    /// parent `Text` or `Widget`, with the default alignment being inherited from the parent.

Reply:
> How about this?

Yeah that works.